pkg/controller/infrastructure/infraflow/context.go

```diff
@@ -75,6 +75,8 @@ const (
 	IdentifierZoneSubnetWorkersRouteTableAssoc = "SubnetWorkersRouteTableAssoc"
 	// IdentifierVpcIPv6CidrBlock is the IPv6 CIDR block attached to the vpc
 	IdentifierVpcIPv6CidrBlock = "VPCIPv6CidrBlock"
+	// IdentifierEgressCIDRs is the key for the slice containing egress CIDRs strings.
```

Comment:
nit:
```suggestion
	// IdentifierEgressCIDRs is the key for the map containing egress CIDRs strings.
```
While not standard maps, the objects it's used with seem more like maps than a slices to me.